addons/mail/models/mail_mail.py

```diff
@@ -576,6 +577,92 @@ def _split_by_mail_configuration(self):
             for batch_ids in tools.split_every(batch_size, record_ids):
                 yield mail_server_id, alias_domain_id, smtp_from, batch_ids
 
+    def _split_by_delayed_batch(self, mail_server):
+        """To not flag personal email servers as spam, we throttle them at X emails / minutes."""
+        if not mail_server or not mail_server.owner_user_id:
+            return self
+
+        # Send the oldest mails in priority if the CRON is late
+        mails = self.browse()
+        notifs = self.env['mail.notification'].search([
```

Comment:
Are we sure access on notification is granted ? Shouldn't we sudo as anyway notification is a technical model and we don't want any acl issue at this point ?